### DISCUSSION TITLE
How to sort a table by alphabet?

### DISCUSSION THREAD

**Discussion by PengJiyuan** [remarkjs/remark#665] [Q&A] [answered]
From:

|Name|Desc|Type|Default|
|---|:---:|:---:|---:|
|className|xxx|fds|fds|
|style|xxx|fd|fd|
|animation|xxx|fds|fds|
|scrollContainer|xxx|fdsfds|fds|

To:

|Name|Desc|Type|Default|
|---|:---:|:---:|---:|
|animation|xxx|fds|fds|
|className|xxx|fds|fds|
|scrollContainer|xxx|fdsfds|fds|
|style|xxx|fd|fd|

**Comment by wooorm** [comment, marked as answer]
by [writing a plugin](https://unifiedjs.com/learn/guide/create-a-plugin/) that [walks the tree](https://unifiedjs.com/learn/recipe/tree-traversal/) to find tables and reorders their rows based on their first child, probably using [`localeCompare`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare)
